### DISCUSSION TITLE
some packages have missing links

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#17908] [General]
after rebuilding everything i noticed that several packages have broken links in the PKGBUILD scripts.
to make matters worse some packages outright refuse to build due to being either outdated or having bugs (example gtksourceview3 or fzf) gtksourceview3 spews loads of undefined references when building. fzf builds the 32 bit package while the enabled 64 bit package refuses to build because it thinks go is 32 bit xD. maybe a spring cleaning is required ?.

**Comment by revelator** [comment]
unfortunatly this is still a problem :/

fzf to.

**Comment by wishesbeast** [comment]
I am also facing the same issue of missing links on [best house cleaners in Ottawa](https://ezi.services/). Let me know when you got the solution. Thanks
